docs/guides/digital-assets/read-asset-data.md

```diff
@@ -52,17 +52,17 @@ In the [**Read Profile Data** guide](../universal-profile/read-profile-data), we
 ```javascript title="read_assets.js"
 
 // Import and network setup
-const Web3 = require("web3");
-const { ERC725 } = require("@erc725/erc725.js");
-require("isomorphic-fetch");
+import Web3 from "web3";
+import { ERC725 } from "@erc725/erc725.js";
+import "isomorphic-fetch";
+import erc725schema from "@erc725/erc725.js/schemas/LSP3UniversalProfileMetadata.json";
 
 // Static variables
 const SAMPLE_PROFILE_ADDRESS = "0xa907c1904c22DFd37FF56c1f3c3d795682539196";
 const RPC_ENDPOINT = "https://rpc.l16.lukso.network";
-const IPFS_GATEWAY = "https://2eff.lukso.dev/ipfs/";
+const IPFS_GATEWAY = "https://cloudflare-ipfs.com/ipfs/";
```

Comment:
I would leave 2eff here, as the script does not work with cloudflare-ipfs.com (infinit load)

@Hugoo